doc/installation.rst

```diff
@@ -252,35 +252,17 @@ In summary:
 
 #. Install the Spyder `requirements`_.
 
-   The recommended and easiest way to do this is with ``conda`` (although experts may prefer ``pip``). In a fresh environment (``conda create -n your-name-here -c conda-forge python=3``, then ``activate`` it), run the following:
+   The recommended and easiest way to do this is with ``conda`` (although experts may prefer ``pip``). To install Spyder in a fresh environment, run the following (omitting the first two commands if you'd like to use an existing one):
```

Comment:
```suggestion
   The recommended and easiest way to do this is with ``conda`` (although experts may prefer ``pip``).
   To install Spyder in a fresh environment, run the following (omitting the first two commands if you'd like to use an existing one):
```

I believe this error was present in the original, but per the docs style guide sentences should each be on their own line.